test/call-test.js

```diff
@@ -1215,7 +1215,7 @@ describe("sinonSpy.call", function() {
                 sinonSpy(myObj, "ouch");
             });
 
-            refute.defined(myObj.ouch);
```

Comment:
Was this removed/changed or did the tests fail here? I'm just curious.

Reply:
It logged a deprecation message.